tfx/components/trainer/executor.py

```diff
@@ -178,6 +185,7 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
         - schema: Schema of the data.
       output_dict: Output dict from output key to a list of Artifacts.
         - output: Exported model.
+        - model_run: Model trainng related outputs (e.g., Tensorboard logs)
```

Comment:
could you also update above -output to -model (we changed key from `output` to `model` but forget to update here )